tiff_2_png/tiff_2_png_converter.py

```diff
@@ -66,7 +68,7 @@ def list_folder_files(path_to_folder: str):
     else:
         logging.error("Folder " + path + " does not exists")
 
-    return file_list, height, width
+    return file_list,width, height
```

Comment:
If I understand correctly it seems like this function will only pass back the width & height of the last image processed. Just wanted to check if that's is intended or if you wanted to pass back lists of widths and heights with the same dimensions of `file_list`.

Reply:
I renamed the variables to make more clear what the function is returning.